consensus/raft/raft.go

```diff
@@ -351,6 +427,11 @@ func (rw *raftWrapper) Peers() ([]string, error) {
 	return ids, nil
 }
 
+// only call when Raft is shutdown
+func (rw *raftWrapper) Clean() error {
+	return os.RemoveAll(rw.dataFolder)
```

Comment:
Note to self: this might cause some breaking changes on the docker test-image, making an issue to check up on and possibly update test-image setup.

Reply:
Hopefully not? I'm not 100% sure, but this should work as before re-starting a removed-peer.